definintion-of-iss-project.md

```diff
@@ -0,0 +1,58 @@
+# Title
+
+The Definition of an InnerSource Software (ISS) Project. We will largely take the ideas that follow
+from the definition of an Open Source Software (OSS) Project. Note, the definition of an OSS Project
+follows from [opensource.com's definition](https://opensource.com/resources/what-open-source). Further,
+[RedHat also supplies a defintion of OSS project](https://www.redhat.com/en/topics/open-source/what-is-open-source).
+
+Our goal here is to mimic the [open source way that opensource.com](https://opensource.com/open-source-way)
+provides. In doing this, we foster collaboration and community inside our enterprises.
+
+## Definition(s)
+
+The following are the components are essential for a project to be declared an InnerSource Software 
+project. Note, the distinction between Open Source and InnerSource is merely
+the network topology surrounding the source control management system instance. If the world at large
+can view the code base, the project is Open Source. On the other hand if any portion of the project's source 
+code lives on a restricted network topology such that it is not available to be read by the world at 
+large then we deem it InnerSource, and refer to it as "private." Also, most OSS projects contain a 
+[license (list taken from opensource.com)](https://opensource.com/law/13/1/which-open-source-software-license-should-i-use).
+The licenses are legal parameters that developers and users adhere to for the development and consumption
+of the project. Luckily we need not worry about licensing because we are concerned with "private" repositories.
+
+We will use the term "population" in the following definitions 
+to reference the user base that has readership based upon the respective network topology. Further, we
+will use the term "owner" to refer to one who has privileges to delete the code base; we will refer to anyone
+who has commits in a repository or documentation or issue list as a "contributor"; we will refer to any person who can 
+push code to the repository a “committer”; we call any person using the repository a “user.” Naturally, the progression 
+of trust follows (most to least trusted) as:
+
+1. Owners (commonly referred to as project committee members)
```

Comment:
I entirely agree with this definition in the Context of OSS.
The context of ISS uses a slightly different, simplified set of terms though, which I think we should keep in the ISS context.

- **Trusted Committers**: This fuses Owners & Committers to a certain degree, there is no real separation in most of all cases.
- **Contributors**: External or Internal to the host project. What it sounds like.
- **Users**: What it sounds like.

I haven't seen the Apache PMC concept in the InnerSource domain yet. There is a role called product owner, however this might not be real match and still needs a bit of work.

The [Learning Path repo](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/03-how-works.asciidoc) has a bit more on those uses of terms.

Reply:
Curious here. In most cases I've found, there are system level distinctions that separate the ability to commit to a repository versus the administrative privileges of the repository (e.g. renaming it or deleting it). I was merely trying to capture that distinction here. That said, if we wish to collapse owner and committer I'm ok with that, but I don't think that it sufficiently describes all of the permissions afforded by most source control systems these days.